src/coreclr/interop/comwrappers.cpp

```diff
@@ -680,6 +678,9 @@ HRESULT ManagedObjectWrapper::QueryInterface(
                     // the same way that would occur if the managed object lacked
                     // an ICustomQueryInterface implementation.
                     break;
+
+                default:
```

Comment:
We are using the fall-through as a fallback. Feel free to use the `[[fallthrough]];` attribute if the compiler is complaining.

Reply:
ok